index.html

```diff
@@ -0,0 +1,191 @@
+                </div>
+            </div>
+            <div class="side-vids">
+                <img src="Essentials/thumbnail.png" class="thumbnail">
+                <div class="side-info">
+                    <a href="" class="title">The Title of the Video</a>
+                    <a href="" class="username">YouTube Creator</a>
+                    <a href="" class="stats">24067 Views • 1 year ago</a>
+                </div>
+            </div>
+            <div class="side-vids">
+                <img src="Essentials/thumbnail.png" class="thumbnail">
+                <div class="side-info">
+                    <a href="" class="title">The Title of the Video</a>
+                    <a href="" class="username">YouTube Creator</a>
+                    <a href="" class="stats">24067 Views • 1 year ago</a>
+                </div>
+            </div>
+            <div class="side-vids">
+                <img src="Essentials/thumbnail.png" class="thumbnail">
+                <div class="side-info">
+                    <a href="" class="title">The Title of the Video</a>
+                    <a href="" class="username">YouTube Creator</a>
+                    <a href="" class="stats">24067 Views • 1 year ago</a>
+                </div>
+            </div>
+            <div class="side-vids">
+                <img src="Essentials/thumbnail.png" class="thumbnail">
+                <div class="side-info">
+                    <a href="" class="title">The Title of the Video</a>
+                    <a href="" class="username">YouTube Creator</a>
+                    <a href="" class="stats">24067 Views • 1 year ago</a>
+                </div>
+            </div>
+            <div class="side-vids">
+                <img src="Essentials/thumbnail.png" class="thumbnail">
+                <div class="side-info">
+                    <a href="" class="title">The Title of the Video</a>
+                    <a href="" class="username">YouTube Creator</a>
+                    <a href="" class="stats">24067 Views • 1 year ago</a>
+                </div>
+            </div>
+            <div class="side-vids">
+                <img src="Essentials/thumbnail.png" class="thumbnail">
+                <div class="side-info">
+                    <a href="" class="title">The Title of the Video</a>
+                    <a href="" class="username">YouTube Creator</a>
+                    <a href="" class="stats">24067 Views • 1 year ago</a>
+                </div>
+            </div>
+            <div class="side-vids">
+                <img src="Essentials/thumbnail.png" class="thumbnail">
+                <div class="side-info">
+                    <a href="" class="title">The Title of the Video</a>
+                    <a href="" class="username">YouTube Creator</a>
+                    <a href="" class="stats">24067 Views • 1 year ago</a>
+                </div>
+            </div>
+        </aside>
+
+    </div>
```

Comment:
Consider using semantic tags for better code readability, user experience, and greater accessibility. Kindly it's better to use a semantic tag like <section> instead of a div.